tests/Integration/Behaviour/Features/Context/CurrencyFeatureContext.php

```diff
@@ -93,7 +93,7 @@ public function thereIsACurrency($currencyName, $currencyIsoCode, $changeRate)
             $currency->save();
         }
         $this->currencies[$currencyName] = $currency;
-        SharedStorage::getStorage()->set($currencyName, $currency);
+        SharedStorage::getStorage()->set($currencyName, (int) $currency->id);
```

Comment:
```suggestion
        $this->getStorage()->set($currencyName, (int) $currency->id);
```

Reply:
I'm not gonna comment all those 😅  You get the idea we should use the class method instead of the static function
This makes me think we should probably deprecate the static function with a trigger notice

Reply:
Note: the idea is not to fix all the calls still present in this PR but at least the ones you modify here I will make a dedicated PR to make all those calls consistent

Reply:
these contexts ain't extending DomainFeatureContext so they don't have $sharedStorage property, thats why static method is used.